plexpy/api.py

```diff
@@ -13,82 +16,108 @@
 #  You should have received a copy of the GNU General Public License
 #  along with PlexPy.  If not, see <http://www.gnu.org/licenses/>.
 
-from plexpy import db, cache, versioncheck, logger, helpers
-
+from plexpy import versioncheck, logger, helpers#, db check why this fails
+import os
 import plexpy
 import json
+import traceback
+import cherrypy
+import re
 from xml.dom import minidom
 
-cmd_list = ['getHistory', 'getLogs', 'getVersion', 'checkGithub', 'shutdown', 'restart', 'update']
+cmd_list = ['getHistory', 'getLogs', 'getVersion', 'checkGithub', 'shutdown',
+            'restart', 'update']
 
 
 class Api(object):
 
     def __init__(self):
-
         self.apikey = None
         self.cmd = None
         self.id = None
-
         self.kwargs = None
-
         self.data = None
-
+        self.msg = None
         self.callback = None
+        self.result_type = 'error'
 
     def checkParams(self, *args, **kwargs):
```

Comment:
The new changes break this function - the initial call from webserve.py expects this function to return something. It now no longer returns anything and none of these errors are going to get caught properly (e.g. you can use any apikey).